content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -276,6 +276,12 @@ Cluster DNS (CoreDNS) will not start up before a network is installed.**
 
 {{< /caution >}}
 
+{{< note >}}
+Currently Calico is the only CNI plugin that the kubeadm project performs e2e tests against.
```

Comment:
small optional tweak
```suggestion
Calico is the only CNI plugin that the kubeadm project performs e2e tests against.
```
because of the very gentle dissuasive style hint from https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future